src/bqlfn.py

```diff
@@ -280,6 +281,35 @@ def bql_column_value_probability(bdb, generator_id, modelno, colno, value,
     r = metamodel.logpdf_joint(
         bdb, generator_id, targets, constraints, modelno)
     return ieee_exp(r)
+
+# XXX This is silly.  We should return log densities, not densities.
```

Comment:
It is a major incompatible change to the semantics of the language.

I wouldn't object to making that change provided we carefully made sure all callers were ready to accept it in bayeslite (including Geweke testing), bdbcontrib, bdb-experiments, &c.  We should probably also run it by Vikash.